src/main/java/uk/ac/ebi/eva/t2d/parameters/validation/T2dPedFileOutputValidator.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2015-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.t2d.parameters.validation;
+
+import org.springframework.batch.core.JobParameters;
+
+import static uk.ac.ebi.eva.t2d.parameters.T2dJobParametersNames.PED_FILE_OUTPUT;
+
+public class T2dPedFileOutputValidator extends SingleKeyValidator {
+
+    public T2dPedFileOutputValidator(boolean mandatory) {
+        super(mandatory, PED_FILE_OUTPUT);
+    }
+
+    @Override
+    protected void doValidate(JobParameters parameters) {
+        // Is a file
```

Comment:
Is this an assert from the programmer that it will be a file, or is this a TODO: do the actual validation?